### PR TITLE
Fix team description styles.

### PR DESCRIPTION
**Pull request by adamjspooner**
@keybase/react-hackers, this is ready for review.

### CODE REVIEW

File: shared/teams/team/header/index.js [hunk 1]
```diff
@@ -61,11 +68,19 @@ const _TeamHeader = (props: Props) => (
...
-            maxWidth: 560,
-          }}
+          className={props.description ? 'hover-underline' : ''}
+          style={collapseStyles([
+            {
+              color: props.description ? globalColors.black_75 : globalColors.black_20,
+              maxWidth: 560,
+              paddingTop: globalMargins.tiny,
+            },
+            platformStyles({
```
Comment (chrisnojima) [inline shared/teams/team/header/index.js]: we have globalStyles.clickable for this

Comment (adamjspooner) [reply shared/teams/team/header/index.js]: Assuming you mean `desktopStyles.clickable`, that only gives me the `pointer`, but I need the `text` cursor when a description is present.

Comment (chrisnojima) [reply shared/teams/team/header/index.js]: ah, we should add that to desktopStyles then you could do
...(props.description ? desktopStyles.editable : desktopStyles.clickable)

Comment (adamjspooner) [reply shared/teams/team/header/index.js]: Will do!

**Comment by adamjspooner** [comment]
@keybase/react-hackers, this is ready for another look.

File: shared/teams/team/header/index.js [hunk 2]
```diff
@@ -61,11 +68,15 @@ const _TeamHeader = (props: Props) => (
       {/* Description */}
       {!props.loading && (props.canEditDescription || props.description) ? (
         <Text
-          style={{
-            paddingTop: globalMargins.tiny,
-            color: props.description ? globalColors.black_75 : globalColors.black_20,
-            maxWidth: 560,
-          }}
+          className={props.description ? 'hover-underline' : ''}
+          style={collapseStyles([
```
Comment (chrisnojima) [inline shared/teams/team/header/index.js]: dont need collapse anymore

**Comment by adamjspooner** [comment]
@keybase/react-hackers, this is ready for another review.

**Pull request merged**